cfme/infrastructure/virtual_machines.py

```diff
@@ -1219,6 +1219,12 @@ def resetter(self, *args, **kwargs):
         self.view.toolbar.reload.click()
 
 
+@navigator.register(Template, 'AnyProviderDetails')
+@navigator.register(Vm, 'AnyProviderDetails')
+class VmAllWithTemplatesDetailsAnyProvider(VmAllWithTemplatesDetails):
```

Comment:
I doubt this is good idea. Inheritance will add some restrictions and can cause issues in navmazing.
@psav, can you share your opinion ?

Reply:
[12:05:22] 18<psav> I think it's OK
[12:05:28] 18<psav> maybe needs a docstring

Reply:
Docstring added as requested